bioconda_utils/build.py

```diff
@@ -261,7 +261,7 @@ def build_recipes(recipe_folder: str, config_path: str, recipes: List[str],
 
     # setup linting
     if do_lint:
-        always_exclude = ('build_number_needs_bump')
+        always_exclude = ('build_number_needs_bump',)
```

Comment:
Fixes error in https://circleci.com/gh/bioconda/bioconda-recipes/65883:
```
13:32:21 BIOCONDA INFO Linting recipe recipes/pylibseq
13:32:21 BIOCONDA ERROR Skipping unknown check s
13:32:21 BIOCONDA ERROR Skipping unknown check m
13:32:21 BIOCONDA ERROR Skipping unknown check n
13:32:21 BIOCONDA ERROR Skipping unknown check e
13:32:21 BIOCONDA ERROR Skipping unknown check p
13:32:21 BIOCONDA ERROR Skipping unknown check d
13:32:21 BIOCONDA ERROR Skipping unknown check _
13:32:21 BIOCONDA ERROR Skipping unknown check i
13:32:21 BIOCONDA ERROR Skipping unknown check b
13:32:21 BIOCONDA ERROR Skipping unknown check u
13:32:21 BIOCONDA ERROR Skipping unknown check l
13:32:21 BIOCONDA ERROR Skipping unknown check r
13:32:21 BIOCONDA ERROR 
```
`set("build_number_needs_bump") == set("smnepd_ibulr")` ;)

Reply:
thanks... that was my bug I think